stack.yaml

```diff
@@ -1,14 +1,9 @@
-resolver: lts-21.17
+resolver: lts-22.43
 
 packages:
 - .
 extra-deps:
```

Comment:
@stschiff I boldly removed some of these fixed versions. I only kept the ones where I remember why we fixed them originally. The tests run through, so I guess this is fine, right?